main.go

```diff
@@ -60,12 +62,30 @@ func YarpNarpHandler(res http.ResponseWriter, req *http.Request) {
 
 func handleGet(res http.ResponseWriter, req *http.Request) {
 	z := Zusage{}
-	if cookie, _ := req.Cookie("nick"); cookie != nil {
-		z.Nick = cookie.Value
-	}
-	z = GetZusage(z.Nick)
-	if cookie, _ := req.Cookie("kommentar"); cookie != nil {
-		z.Kommentar = cookie.Value
+
+	if c, _ := req.Cookie("yarpnarp"); c != nil {
+		keks, err := base64.StdEncoding.DecodeString(c.Value)
+		if err != nil {
+			log.Println(err)
+			return
+		}
+
+		cookie := struct {
```

Comment:
I would define a global Cookie-type, so that you don't have to change it in two places at once.